networkx/tests/test_convert_scipy.py

```diff
@@ -1,9 +1,8 @@
 import pytest
 
 np = pytest.importorskip("numpy")
+npt = pytest.importorskip("numpy.testing")
 sp = pytest.importorskip("scipy")
```

Comment:
Does `pytest.importorskip` load the subpackages, or does this also need a `sp.sparse = pytest.importorskip("scipy.sparse")`, since `sparse` is used in some of the tests below?